techniques/html/H39.html

```diff
@@ -40,15 +40,14 @@ <h2>Tests</h2>
 				<h3>Procedure</h3>
         <p>For each data table:</p>
         <ol>
-					<li>Check that the table has content that is presented as a table caption.</li>
           <li>Check that the table includes a <code class="language-html">caption</code> element.</li>
```

Comment:
Check if, rather than check that? Because the rest is condtional on there being a caption at all?

Reply:
Since this PR is not proposing to alter that phrasing in this PR, I don't think it's appropriate to tackle here. My suggestion is to add this as another element in https://github.com/w3c/wcag/pull/3915